src/helpers/installer.ts

```diff
@@ -0,0 +1,102 @@
+import puppeteer, { BrowserFetcher, RevisionInfo } from 'puppeteer-core';
+import ProgressBar from 'progress';
+import preLogger from './logger';
+
+let browserFetcher: BrowserFetcher;
+
+// Override current environment proxy settings with npm configuration, if any.
+const NPM_HTTPS_PROXY =
+  process.env.npm_config_https_proxy || process.env.npm_config_proxy;
+const NPM_HTTP_PROXY =
+  process.env.npm_config_http_proxy || process.env.npm_config_proxy;
+const NPM_NO_PROXY = process.env.npm_config_no_proxy;
```

Comment:
We could create differenct constant to declare `process.env.npm_config_proxy` to make it reusable like the constant `NPM_NO_PROXY` but it's not mandatory.

Reply:
The constants and environment variables here mimic exactly how puppeteer and puppeteer-core packages are configured. I don't prefer to change anything on there.